pkg/api/plugins.go

```diff
@@ -38,12 +38,12 @@ func (hs *HTTPServer) getPluginContext(pluginID string, user *models.SignedInUse
 		if err != models.ErrPluginSettingNotFound {
 			return pc, errutil.Wrap("Failed to get plugin settings", err)
 		}
+		decryptedSecureJSONData = make(map[string]string)
```

Comment:
How about we always init it at line 33 instead? 
line 47 could overwrite later.

Reply:
Thanks. Added that improvement